.github/workflows/pull-request-container.yml

```diff
@@ -56,6 +59,19 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Configure AWS Credentials
+        if: ${{ env.AWS_CI_ROLE != '' }}
```

Comment:
Is this check necessary? It's unconditionally included above, and secrets aren't scoped, so the only way this could happen is if we set it to the empty string (or possibly if we remove the secret).

Reply:
I believe it was a protection for forks, so if you fork the repository the builds don't fail.